bindings/pyroot_experimental/cppyy/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+if(MSVC)
+  set(CMAKE_SHARED_LIBRARY_SUFFIX ".dll")
+else()
+  set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
```

Comment:
Yes, cppyy expects its libraries to have `.dll`extension on Windows and `.so` otherwise (including Mac). E.g. this is cppyy right before loading `libcppyy_backend`:
```python
if 'win32' in sys.platform:
    soext = '.dll'
else:
    soext = '.so'
```